conferences/abstracts/RSECon19-BinderHub-workshop-application.md

```diff
@@ -0,0 +1,96 @@
+# Research Software Engineering Conference 2019
+
+* Conference website: https://rse.ac.uk/conf2019/
+* Application form: https://docs.google.com/forms/d/e/1FAIpQLScVml19n44QfuOfjuaRlJUtXNMih-X4xUrNjlr8XJVHeyTyQg/viewform
+* Location: University of Birmingham
+* Date: 17th - 19th September 2019
+
+### Workshop Length
+
+180 mins (to allow time for Kubernetes cluster deployment and an introduction to reproducible research and BinderHub).
+
+### Title
+
+"Build a BinderHub for hosting Reproducible Software in the Cloud"
+
+### Authors
+
+* [Sarah Gibson](https://www.turing.ac.uk/people/researchers/sarah-gibson), ([The Alan Turing Institute](https://www.turing.ac.uk/)) on behalf of **[The Turing Way](https://github.com/alan-turing-institute/the-turing-way) Collaboration**.
+
+### Abstract
+
+_(1250 characters max, current count: 1244 [please update if you edit])_
+
+BinderHub is a Cloud-based, multi-user server technology that captures the computing environment of a GitHub repository within a Docker image.
+It then provides an interactive browser from which code can be run in the environment under which it was developed without placing installation responsibilities on the user.
+A unique URL to this browser is generated making the software easily sharable with collaborators.
+BinderHubs can be built anywhere in the world and are Cloud-neutral.
+Building a locally hosted BinderHub is advantageous as it would facilitate the use of private repositories and make available a greater variety of computing resources - facilities which are not currently available on the public Binder service.
+
+By harnessing the power of Cloud computing and building a BinderHub with expanded capabilities, scientific analyses can become more accessible and transparent to the wider community.
+BinderHubs can showcase the reproducibility of software as well as being a valuable teaching or demonstration tool.
+
+In this workshop, attendees will learn how to deploy a BinderHub on Microsoft Azure and connect with others who have or are interested in doing so.
+Attendees should be familiar with Azure Cloud computing and the command line.
```

Comment:
I think this is asking too much. It's making the workshop too specific.

Also, now that I've read the section below, I think you can drop this whole sentence.

```suggestion
```

Reply:
There is a thread in RSE Slack that says the "Audience" and "Outcomes" sections won't be published online, but I feel they're quite critical sections for a potential attendee to understand what they're going to get out of the workshop. Happy to reword but I'm not entirely comfortable dropping completely.

Reply:
I've changed it to "Attendees should be comfortable with the command line." I mentioned Azure in the previous sentence, so hopefully this comes across less strict.